src/index.js

```diff
@@ -0,0 +1,31 @@
+import './style.css';
+import render from './populateui.js';
+import AddToDoList from './list.js';
+
+const taskList = new AddToDoList();
+render(taskList);
+
+const addTodoBtn = document.querySelector('.add-btn');
+addTodoBtn.addEventListener('click', () => {
+  const id = `id${Math.random().toString(16).slice(2)}`;
+  const description = document.querySelector('.input-list').value.trim();
+  const completed = false;
+  const index = taskList.list.length + 1;
+
+  const newTask = {
+    id,
+    description,
+    completed,
+    index,
+  };
+  if (description) {
+    taskList.addTask(newTask);
+    render(taskList);
+  }
+});
+
+const clearBtn = document.querySelector('.clear-btn');
+clearBtn.addEventListener('click', () => {
+  taskList.clearCompletedTask();
+  render(taskList);
```

Comment:
![requirements](https://user-images.githubusercontent.com/93039359/184006204-ef82fbb8-f27c-404c-8a22-c7dd4359b438.png)

- [ ] You are not following all the requirements. Please follow the missing requirements as shown in the image above. :blush:
- Please add some todos in an array of objects
- Then iterate over each todo in the array of objects
- Then append them in HTML

Reply:
Kindly check list.js they are implemented they're  and the name is not todos is a tasklist for me